applications/ipc_radio/README.rst

```diff
@@ -63,8 +63,10 @@ Application
 You can set the supported radio configurations using the following Kconfig options:
 
 * :kconfig:option:`CONFIG_IPC_RADIO_BT` - For the Bluetooth Low Energy serialization.
+* :kconfig:option:`CONFIG_IPC_RADIO_BT_RPC` - For the Bluetooh ost API serialization.
```

Comment:
The distinction between rpc and hci is described on line 70. Maybe you can move this line (and add one about hci_ipc option) there after line 70?

Reply:
Changed, added list of choices available in https://github.com/nrfconnect/sdk-nrf/blob/fb4898139020f3f1f7297571ab6b6364d7d179ff/applications/ipc_radio/Kconfig#L22-L41